runtime/server/server.go

```diff
@@ -243,9 +243,15 @@ func timeoutSelector(service, method string) time.Duration {
 	if service == "rill.runtime.v1.RuntimeService" && (strings.Contains(method, "Trigger") || strings.Contains(method, "Reconcile")) {
 		return time.Minute * 30
 	}
+
 	if service == "rill.runtime.v1.QueryService" {
 		return time.Minute * 5
 	}
+
+	if method == WatchFilesMethod {
```

Comment:
Should we refactor this func to pass in the full method name, and then use `runtimev1.RuntimeService_WatchFiles_FullMethodName` (and related constants) instead?